pkg/columns/columns.go

```diff
@@ -502,6 +502,31 @@ func GetFieldFuncExt[OT any, T any](column ColumnInternals, raw bool) func(entry
 	}
 }
 
+// GetFieldAsArrayFunc returns a helper function to access an array of type OT of a struct T
+// without using reflection. It does not differentiates between direct members of the struct and
+// members of embedded structs.
+func GetFieldAsArrayFunc[OT any, T any](column ColumnInternals) func(entry *T) []OT {
+	l := column.(*Column[T]).Type().Len()
+	s := column.(*Column[T]).Type().Elem().Size()
```

Comment:
`RawType` should be better suited here, as `Type` could be `string` if an extractor has been set.

Reply:
Thanks, I also added a test for this where an extractor gets set